homeassistant/helpers/script.py

```diff
@@ -545,6 +550,41 @@ def _finish(self):
         super()._finish()
 
 
+async def _async_stop_scripts_after_shutdown(hass, point_in_time):
+    """Stop running Script objects started after shutdown."""
+    running_scripts = [
+        script for script in hass.data["Scripts"] if script["instance"].is_running
```

Comment:
Can we extract this into a constant `DATA_SCRIPTS` and make the value `helpers.script` ?